source/sessions/driver-sessions.rst

```diff
@@ -945,6 +954,27 @@ topologies.  It MUST NOT be run against a standalone server.
     * Capture the command sent to the server
     * Assert that the command sent to the server does not have an ``lsid`` field
 
+6. Client-side cursor that exhausts the results on the initial query immediately returns the implicit session
+to the pool.
+    * Insert two documents into a collection
```

Comment:
I don't understand the significance of *two* documents.

The important thing is that the entire result set is returned in the first batch right?

Reply:
It's a response to Jesse's concern that some drivers don't send the initial find operation until the application starts iterating.  This lets a test iterate past the first document to force the find operation, then make the assertion.